### ISSUE TITLE
Int64 decompiler System.NotImplementedException exception

### ISSUE THREAD

**Issue by LukeSaward1** [krzys-h/UndertaleModTool#590]
![image](https://user-images.githubusercontent.com/55454721/134766687-de808044-88af-4777-a15e-dad884467c68.png)
I get the above error when putting this code into gml_Object_DEVICE_MENU_Step_0 code entry:
```  
global.isTimerRunning = 1
if (global.isTimerRunning == 1)
{
    global.timeStart = get_timer()
}
```
using the timer code from the Undertale RTA timer mod.

Is there any reason why this would happen?

**Comment by Archie-osu** [comment]
It may actually be a compiler / decompiler issue, as GMS 2.3 is not yet fully supported - at this point in time, it is recommended to edit assembly, and not decompiled code.

**Comment by LukeSaward1** [comment]
I wish I knew assembly, I guess I need to learn it if I want to mod DRC2 rn.
Kinda would have been better if toby went with the version of gms before 2.3, although I don't know if that would have made it easier to mod; as I don't know if the exceptions are caused by GMS2.3 or if they would also appear on GMS2 versions before 2.3

**Comment by Archie-osu** [comment]
I'll try implementing a potential fix myself - where exactly inside gml_Object_DEVICE_MENU_Step_0 have you added the supplied code?

**Comment by Archie-osu** [comment]
![image](https://user-images.githubusercontent.com/61205459/134767090-11eadc8e-4b16-475b-a318-1cf29ce0dba6.png)
Was it like this, at the top?

**Comment by LukeSaward1** [comment]
It was at the bottom in the
```
if(START_GAME == 1)
{
}
```
block I think

**Comment by Grossley** [comment]
Fixed in 62fe347a0017476505f5e0415c2b5f549f690a17
